app/models/like.rb

```diff
@@ -0,0 +1,13 @@
+class Like < ApplicationRecord
+  belongs_to :author, class_name: 'User'
+  belongs_to :post
+
+  after_create :update_post_likes_counter
+  after_destroy :update_post_likes_counter
+
+  private
+
+  def update_post_likes_counter
+    post.update_posts_counter
+  end
```

Comment:
- [ ] Same here, we need to increment the value of the `likes_counter` property of the post. Please notice we have two different attributes (`likes_counter` and `commnets_counter`) and every time a new comment enters we need to update the comment counter and when there is a new like we need to update the likes counter.